### ISSUE TITLE
Crash at start MO2 alpha 3

### ISSUE THREAD

**Issue by LeFlemard** [TanninOne/modorganizer#486]
Hi!
After doing the initial first prompt (the one choosing the type of instance), MO immediately crash during the apparition of the logo. I can send the dmp file if you need...
Thanks

**Comment by DeadMor0z** [comment]
If you have ModOrganizer.ini file from previous alpha, overwrite new verion with it.

**Comment by PRieST47** [comment]
I have the same problem, overwriting the ini didn't work for me

**Comment by DeadMor0z** [comment]
Do you have portable or regular install?

**Comment by PRieST47** [comment]
I think I did regual, but even if I delete every file genereted from MO2, redownload it and 'reinstall' it I won't get a prompt to chose, MO2 is just crashing directly.
I am not able to switch between portable and regular right now

**Comment by DeadMor0z** [comment]
Well, if you have ModOrganizer folder appearing in (SystemDrive):\Users\_your_user_\AppData\Local, then you have a regular version and should overwrite ModOrganizer.ini there, not in MO install folder.

**Comment by PRieST47** [comment]
Yeah, did it just like you said, but it didn't help me

I just tried to rename 'bsa_extractor.dll' into 'bsa_extractor.!dll' like it is mentioned in another thread.
And this worked fine for me. Now I can start MO2 alpha 3 and it works.

**Comment by PRieST47** [comment]
Believe it or not, I tried this yesterday a few times and it did't work

Now, I thought I'll give it a last try...and now it just started perfectly...
Sometimes I just hate IT and everything which has to do with it xD

**Comment by DarkenRahhl** [comment]
Same issue for me. Downloaded alpha 3 and tried to launch it as admin etc, and it just crashes directly

How do I find my MO ini's?

**Comment by DeadMor0z** [comment]
If you chosed portable install - lok for it in MO dir, otherwise - look in (SystemDrive):\Users\_your_user_\AppData\Local\ModOrganizer.

**Comment by Loganbacca** [comment]
I also ran into this issue. I checked the ModOrganizer.ini file and it was pointing to the New Vegas install files (even though I don't have New Vegas installed). This occurred after doing a clean install and selecting the non-portable option.

Editing the ModOrganizer.ini file and removing everything and replacing it with:

	[General]
	gamePath=C:\\Program Files (x86)\\Steam\\steamapps\\common\\Fallout 4

(or wherever your Fallout 4 folder is) worked for me.

**Comment by noctred** [comment]
Had the same issue and also noticed the New Vegas path in ModOrganizer.ini.

However, for me, just replacing it with the Fallout 4 path didn't work because every time I started MO, it would overwrite the changes I made and revert to an invalid game path, causing an instant crash once again.

What ultimately fixed it for me was changing the gamePath, making the file read-only, starting MO, exiting MO, and then removing the read-only flag from the file. From that point forward, MO starts without crashing.

HTH.

**Comment by ogrotten** [comment]
(copying my entry from #490 to here to aid discussion.)

Based on advice from a separate issue, I went with a fresh unzip, as opposed to the overwrite unzip I'd done earlier.

I had chosen "single install". MO Crashed after that prompt. Here's the dump.
[ModOrganizer.exe.dmp.zip](https://github.com/TanninOne/modorganizer/files/88656/ModOrganizer.exe.dmp.zip)

I deleted and re-un-zipped the dir and started it again. This time I answered "multiple install". crash again, I don't ahve a dump.

I took the ini from 2.0.2a and overwrote the ini in the multi-install dir.After starting, MO did NOT crash as it had before, but instead gave me an error message that it couldn't find the directory for the profile that i had been using in 2.0.2a and the hybrid.

I then deleted the directory and re-un-zipped again. This time, straight to crash without a prompt for what type of install I want to do.

I deleted the directory buried in the windows profile, started it again, it crashed without prompting me. I dug around in the registry and deleted HKEY_CLASSES_ROOT\...\Tannin.

Upon restart, I was prompted for type of install. I chose Portable and it crashed. I copied over the old .ini and the old profiles directory and now it ran without problem . . . except, of course, the mods list was empty.

I again copied /profiles (overwriting), along with /mods and /downloads. It ran with everything the way i had it in v2.0.2a install, and the v2.0.2a install with 2.0.3 unzipped over it. The exception being that the game now bombs on start... but that's a different problem altogether.

I could probably recreate any of the above conditions on purpose if you have questions or want me to try something.

**Comment by DarkenRahhl** [comment]
What exactly am I supposed to replace 

[General]
gameName=New Vegas
gamePath=D:\\Program Files (x86)\\Mod tools\\ModOrganizer2

with?

That gamepath is correct, I think. Here is the game path 
D:\Program Files (x86)\Steam games\SteamLibrary\steamapps\common\Fallout 4

**Comment by michaelrw10** [comment]
@DarkenRahhl just replace the incorrect gamePath location with the correct one. Then delete the entire gameName=New Vegas  line
save and then try to launch the program again. (some people report having to set the ini file to read-only after they make the changes to it, but i did not have to)

Me personally, I am having problems getting the program to ask me what type of installation I want (either single game with multiple installation instances, or one MO installation that serves all of the games). I think the registry must have the values set so that it is not asking me again when I install a clean MO.. I am searching registry for the value mentioned above, but so far I havent found the culprit.

**Comment by DarkenRahhl** [comment]
Michael how is the sentence supposed to look like?

[General]
gamePath=D:\Program Files (x86)\Steam games\SteamLibrary\steamapps\common\Fallout 4
gameName=Fallout 4 

I have no idea, really... Sorry for being dumb hehe

**Comment by ogrotten** [comment]
@DarkenRahhl I believe the `\` need to be escaped, so each of them would need to be `\\`

**Comment by Leer0x** [comment]
To fix this issue
I did like was said above: changed my gamePath to my correct gamePath (C:\Games\Fallout 4 BETA)
but instead, has to add a second backslash for some reason, so I did that, and it looks like this:
C:\\Games\\Fallout 4 BETA
I did this and it seems to start up

**Comment by ogrotten** [comment]
did you mean that it looks like `C:\\Games\\Fallout 4 BETA`?

**Comment by Leer0x** [comment]
yes i wrote this lol.
When I press edit comment it says C:\\Games\\Fallout 4 Beta WITH 2 backslashes if that doesn't go in again

**Comment by TanninOne** [comment]
was resolved in alpha4
